### PR TITLE
[Release-63, Tests.] Get altitude integration test for all mwm features.

### PR DESCRIPTION
**Pull request by bykoianko**

### CODE REVIEW

**Comment by bykoianko** [comment]
@ygorshenin or @mpimenov PTAL

File: routing/routing_integration_tests/get_altitude_test.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    f.ParseGeometry(FeatureType::BEST_GEOMETRY);
+    size_t const pointsCount = f.GetPointsCount();
+    if (pointsCount == 0)
+      return;
+
+    TAltitudes altitudes = altitudeLoader->GetAltitudes(id, pointsCount);
+    TEST(!altitudes.empty(),
+         ("Empty altidude vector. MWM:", countryId, ", feature id:", id, ", altitudes:", altitudes));
+
+    for (auto const alitude : altitudes)
```
Comment (mpimenov) [inline routing/routing_integration_tests/get_altitude_test.cpp]: s/alitude/altitude/g

File: routing/routing_integration_tests/get_altitude_test.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    {
+      TEST_EQUAL(my::clamp(alitude, minAltitudeMeters, maxAltitudeMeters), alitude,
+                 ("Unexpected altidude. MWM:", countryId, ", feature id:", id, ", altitudes:", altitudes));
```
Comment (mpimenov) [inline routing/routing_integration_tests/get_altitude_test.cpp]: s/altidude/altitude/g

File: routing/routing_integration_tests/get_altitude_test.cpp [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+#include "std/string.hpp"
+#include "std/unique_ptr.hpp"
+#include "std/utility.hpp"
+
+namespace
+{
+using namespace feature;
+
+void TestAltitudeOfAllMwmFeatures(string const & countryId, TAltitude const minAltitudeMeters,
```
Comment (mpimenov) [inline routing/routing_integration_tests/get_altitude_test.cpp]: s/minAltitudeMeters/altitudeLowerBoundMeters/
и
s/maxAltitudeMeters/altitudeUpperBoundMeters/

как вариант

File: routing/routing_integration_tests/get_altitude_test.cpp (hunk 1 above)
Comment (bykoianko) [inline routing/routing_integration_tests/get_altitude_test.cpp]: fixed

File: routing/routing_integration_tests/get_altitude_test.cpp (hunk 2 above)
Comment (bykoianko) [inline routing/routing_integration_tests/get_altitude_test.cpp]: fixed.

**Comment by bykoianko** [comment]
@ygorshenin or @mpimenov PTAL

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**